src/gbt_convertors.pyx

```diff
@@ -70,9 +70,13 @@ def get_gbt_model_from_lightgbm(model: Any) -> Any:
             continue
 
         # add root
+        feat_val = struct["threshold"]

```

Comment:
minnor: struct bad name, github even emphasizes as a keyword

Reply:
renamed to `tree_struct` :D